docs/docs/faq.mdx

```diff
@@ -157,7 +157,8 @@ Note: Tilde (~) in paths refers to your user's home directory.
 <TabItem value="powershell">
 
 ```powershell
-# ~/Documents/WindowsPowerShell/profile.ps1
+# ~\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1 (PowerShell v5.1 & below)
```

Comment:
It's actually smarter to reference `$profile`

Reply:
yup. ok.

Reply:
pushed a new commit. @JanDeDobbeleer